Common/Prey/GameDll/ark/iface/IArkAbilityListener.h

```diff
@@ -7,8 +7,8 @@
 class IArkAbilityListener // Id=8013975 Size=8
 {
 public:
-	virtual void OnAbilityAdded(uint64_t _abilityID);
-	virtual void OnBecomeAlien();
+	virtual void OnAbilityAdded(uint64_t _abilityID) = 0;
+	virtual void OnBecomeAlien() = 0;
```

Comment:
The interface actually has a default implementation that does nothing. You can see the function references in the file below

```cpp
static inline auto FOnAbilityAdded = PreyFunction<void(IArkAbilityListener *const _this, uint64_t _abilityID)>(0xA13080);
static inline auto FOnBecomeAlien = PreyFunction<void(IArkAbilityListener *const _this)>(0xA13080);
```